distribution/docs/src/main/resources/ManContents.adoc

```diff
@@ -2301,6 +2301,18 @@ Event event = new Event("ddf/notification/catalog/downloads", properties);
 eventAdmin.postEvent(event);
 ----
 
+=== Configuring system.properties
+
```

Comment:
May want to change the description based upon where this section ends up in the document.